magento/magento-cloud
# MCLOUD-5321: Update Cloud templates

### Description
Added command into composer.json for auto-generation `.magento.env.md` file

### Manual testing scenarios

1. Run composer install/update
2. Check that `.magento.env.md` was created in the root directory

### Contribution checklist
